packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1716,6 +1708,19 @@ class ThemeData with Diagnosticable {
   )
   final AndroidOverscrollIndicator? androidOverscrollIndicator;
 
+  /// The color used to highlight the active states of toggleable widgets like
+  /// [Switch], [Radio], and [Checkbox].
+  Color get toggleableActiveColor => _toggleableActiveColor!;
```

Comment:
Same here, why does this move? Also, I think this should be deprecated already... it might be an oversight from another change. In general, since this PR is for primaryX, anything else should be done in a separate change.

Reply:
`ThemeData` is now strictly organized, these were not re-ordered in past PRs, as noted in the PR description.

Reply:
Right, does all of this need to happen in one PR?

Reply:
I don't think it needs to happen in separate PRs, we often fix things as we go. I can edit the title to reflect that